pkgs/by-name/ls/lsp-ai/package.nix

```diff
@@ -0,0 +1,75 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  cmake,
+  openssl,
+  zlib,
+  perl,
+}:
+
+rustPlatform.buildRustPackage {
+
+  pname = "lsp-ai";
+  version = "0.7.0";
+  src = fetchFromGitHub {
+    owner = "SilasMarvin";
+    repo = "lsp-ai";
+    rev = "476a13ed2dfec064cc4bc8530ba432f30a9dfe9e";
```

Comment:
It isn't broken. I used `nix-prefetch-github SilasMarvin lsp-ai --rev release/v0.7.0 --nix`

But I can add that syntax

Reply:
Reverting this change, introduced vetting issue:

```bash
       at /home/runner/work/nixpkgs/nixpkgs/pkgs/by-name/ls/lsp-ai/package.nix:19:25:

           18|     repo = "lsp-ai";
           19|     rev = "refs/tags/v${version}";
             |                         ^
           20|     hash = "sha256-5chIz9uIKmwnZG1HjEjqZqoWi3aTmh51r5YJD21u2OE=";
- Nix evaluation failed for some package in `pkgs/by-name`, see error above
This PR introduces the problems listed above. Please fix them before merging, otherwise the base branch would break.
To run locally: ./ci/nixpkgs-vet.sh master https://github.com/NixOS/nixpkgs.git
If you're having trouble, ping @NixOS/nixpkgs-vet
```

Reply:
fixed